### PR TITLE
[WFX-455] Update AUS logic, so latest version receives no update and …

### PR DESCRIPTION
**Pull request by MrAlex94**
…previous versions receive latest update.

### CODE REVIEW

File: .github/workflows/production.yml [hunk 1]
```diff
@@ -59,13 +59,32 @@ jobs:
...
-            update_loc=":s3:aus/update/staging/${{ env.CHANNEL }}/${{ inputs.DISPLAY_VERSION }}/"$j"/update.xml"
-            mkdir -p aus_tmp/$j/
-            rclone copyto :s3:aus/update/staging/${{ env.CHANNEL }}/${{ inputs.DISPLAY_VERSION }}/"$j"/update.xml ./aus_tmp/$j/update.xml
-            sed -i 's/staging/releases/g' ./aus_tmp/$j/update.xml
-            sed -i 's/staging/releases/g' ./aus_tmp/$j/update.xml
-            sed -i 's/staging/releases/g' ./aus_tmp/$j/update.xml
-            rclone moveto ./aus_tmp/$j/update.xml :s3:aus/update/production/${{ env.CHANNEL }}/${{ inputs.DISPLAY_VERSION }}/"$j"/update.xml --dry-run
+            mkdir -p aus_tmp/"$j"/
+            rclone copyto :s3:aus/update/staging/${{ env.CHANNEL }}/${{ inputs.DISPLAY_VERSION }}/"$j"/update.xml ./aus_tmp/"$j"/update.xml
+            sed -i 's/staging/releases/g' ./aus_tmp/"$j"/update.xml
```
Comment (adamp01) [inline .github/workflows/production.yml]: This seems to just be doing the same thing 3 times?

File: .github/workflows/production.yml [hunk 2]
```diff
@@ -59,13 +59,32 @@ jobs:
           os=(WINNT_x86_64 Linux_x86_64 Darwin_x86_64-aarch64)
           for j in ${os[@]}
```
Comment (adamp01) [inline .github/workflows/production.yml]: We should make a note to make the variable names logical here, not entirely sure why I ever decided to set this as j

File: .github/workflows/production.yml [hunk 3]
```diff
@@ -59,13 +59,32 @@ jobs:
...
-            sed -i 's/staging/releases/g' ./aus_tmp/$j/update.xml
-            sed -i 's/staging/releases/g' ./aus_tmp/$j/update.xml
-            rclone moveto ./aus_tmp/$j/update.xml :s3:aus/update/production/${{ env.CHANNEL }}/${{ inputs.DISPLAY_VERSION }}/"$j"/update.xml --dry-run
+            mkdir -p aus_tmp/"$j"/
+            rclone copyto :s3:aus/update/staging/${{ env.CHANNEL }}/${{ inputs.DISPLAY_VERSION }}/"$j"/update.xml ./aus_tmp/"$j"/update.xml
+            sed -i 's/staging/releases/g' ./aus_tmp/"$j"/update.xml
+            sed -i 's/staging/releases/g' ./aus_tmp/"$j"/update.xml
+            sed -i 's/staging/releases/g' ./aus_tmp/"$j"/update.xml
+          done
+          for i in $(rclone lsf --dirs-only :s3:aus/update/production/${{ env.CHANNEL }} | awk -v RS= '{$1=$1}1' | sed 's|/||g')
```
Comment (adamp01) [inline .github/workflows/production.yml]: We should change the `i` over to something logical, like `version` or something. Could be handy to have a comment stating what the `awk | sed` is changing the output from and to.

Comment (MrAlex94) [reply .github/workflows/production.yml]: Aye, good shout on the comment. Will add now.

**Pull request merged**